test/package.js

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs';
+import test from 'ava';
+import index from '../';
+
+test.cb('Every rule is defined in index file', t => {
+	fs.readdir('../rules/', function (err, files) {
+		t.ifError(err);
+
+		var rules = files.filter(file => file.indexOf('.js') === file.length - 3);
```

Comment:
Feel free to do it if you care enough, but it will be moot in a couple of months.